packages/@blockprotocol/type-system-node/src/index_bg.wasm.d.ts

```diff
@@ -0,0 +1,19 @@
+/* tslint:disable */
```

Comment:
Should we add `.gitattributes` for the auto-generated files similarly to what we did for #928 (this goes for the other package as well)?

Reply:
We can yes. What do we get from doing so in this case?

Reply:
They'll be suppressed in diffs and excluded from the language statistics on the GitHub repo

Reply:
I'm happy to add this if you also think it's a good idea @kachkaev

Reply:
Yep, let's do that. Why not?

Reply:
That's been [done](https://github.com/blockprotocol/blockprotocol/pull/503/files/15ef69e4a87fea093bae318405001dab2f02f9c4..994b6b238ed95057452f443ec3272b14e0a5af49)

I assume that's what you had in mind @thehabbos007?